src/layouts/CoreLayout/CoreLayout.scss

```diff
@@ -1,2 +1,11 @@
 .mainContainer {
 }
+section:first-child {
```

Comment:
I might want to move this into `CoreLayout.scss`. If the inner one needs to be modified, lets change the `mainContainer` class.